### ISSUE TITLE
ConsumerGroup only comsume one partition

### ISSUE THREAD

**Issue by emmaccacca** [SOHU-Co/kafka-node#882]
## Questions?

I use HighLevelProducer to produce keyed message and then I use ConsumerGroup to consume messages. But I only get message of one partition every time I restart my service;

### Environment

* Node version: 8.9.4
* Kafka-node version: 2.11
* Kafka version: 1.0

### For specific cases also provide
* Number of Brokers: 3
* Number partitions for topic: 4

### Include Sample Code to reproduce behavior

```js
                producer.send([{
                    topic: 'topic',
                    messages: new KeyedMessage('key', 'message')
                }], (err, data) => {
                    if (err) {
                        console.error(err);
                    }
                });

        const consumerOptions = {
            kafkaHost,
            autoCommit: false,
            fromOffset: 'earliest'
        };

        const consumer = new Consumer(Object.assign({id: '1'}, consumerOptions), ['topic']);
```

### Include output with [Debug](https://github.com/SOHU-Co/kafka-node#how-do-i-debug-an-issue) turned on

Thanks for your contribution!

**Comment by aikar** [comment]
As I responded on https://github.com/SOHU-Co/kafka-node/issues/904 - 
Considering your import is named "Consumer", and not "ConsumerGroup", I assume you imported from lib/consumer.js

If this is the case, you need to use ConsumerGroup (lib/consumerGroup.js) instead, if you want all partitions to be automatically assigned.
Consumer requires manually specifying partitions.

@hyperlink I can see why this is a source of confusion considering the docs list Consumer first, but still uses a group ID in config.

Maybe need to update docs to say 'You probably want <ConsumerGroup>'

**Comment by hyperlink** [comment]
You're right @aikar there's certainly improvement opportunities in the docs.
